### DISCUSSION TITLE
Updated Documentation

### DISCUSSION THREAD

**Discussion by primetomas** [Keyfactor/ejbca-ce#399] [Announcements]
The container documentation was updated with exhaustive lists of environment variables and directories of importance. Making it easier to fully understand how to use the container in a larger setting.

**Comment by MrWildanMD** [comment]
is the documentation is updated alongside the current ejbca-ce version?
thanks

**Reply by primetomas** [reply]
It's all in here. https://hub.docker.com/repository/docker/keyfactor/ejbca-ce/general
But now, I will actually change it and instead link to the helm chart repository that lists all the environment variables, so we only have it in one (or a few :-)) places. https://github.com/Keyfactor/ejbca-community-helm

**Comment by aleloco09** [comment]
I have not been able to reach your documentation website https://doc.primekey.com/ for a few days now. 
Has it been shut down?

**Reply by primetomas** [reply]
doc.primekey.com is temporarily down due to IT issues.

**Reply by djlambert** [reply]
Any update on when the documentation site will be back up? Or is it available elsewhere (in a repo maybe)?

**Reply by primetomas** [reply]
Backup location here: https://download.primekey.com/docs/EJBCA-Enterprise/latest/EJBCA_8.1_Documentation.html

**Reply by primetomas** [reply]
https://doc.primekey.com/ is up again
